<launch>
    <node name="navigation_client" pkg="igvc_navigation" type="navigation_client" output="screen">
        <param name="reading_from_file" value="true"/>
        <param name="waypoint_radius" value="3"/>
    </node>
</launch>